[2025-02-26T21:11:41.293+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: api_to_db_dag.create_station_informations_table manual__2025-02-26T21:11:40.889547+00:00 [queued]>
[2025-02-26T21:11:41.302+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: api_to_db_dag.create_station_informations_table manual__2025-02-26T21:11:40.889547+00:00 [queued]>
[2025-02-26T21:11:41.302+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-02-26T21:11:41.302+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2025-02-26T21:11:41.302+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-02-26T21:11:41.310+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_station_informations_table> on 2025-02-26 21:11:40.889547+00:00
[2025-02-26T21:11:41.313+0000] {standard_task_runner.py:55} INFO - Started process 331 to run task
[2025-02-26T21:11:41.316+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_to_db_dag', 'create_station_informations_table', 'manual__2025-02-26T21:11:40.889547+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpgknpgpp7']
[2025-02-26T21:11:41.317+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask create_station_informations_table
[2025-02-26T21:11:41.363+0000] {task_command.py:389} INFO - Running <TaskInstance: api_to_db_dag.create_station_informations_table manual__2025-02-26T21:11:40.889547+00:00 [running]> on host 31a764ff7239
[2025-02-26T21:11:41.403+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_to_db_dag
AIRFLOW_CTX_TASK_ID=create_station_informations_table
AIRFLOW_CTX_EXECUTION_DATE=2025-02-26T21:11:40.889547+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-26T21:11:40.889547+00:00
[2025-02-26T21:11:41.403+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS velib_station_details (
            station_id BIGINT PRIMARY KEY, 
            capacity INT,                   
            lat DOUBLE PRECISION,          
            lon DOUBLE PRECISION,           
            name VARCHAR(255)               
        );
        
[2025-02-26T21:11:41.409+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-02-26T21:11:41.525+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-02-26T21:11:41.529+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS velib_station_details (
            station_id BIGINT PRIMARY KEY, 
            capacity INT,                   
            lat DOUBLE PRECISION,          
            lon DOUBLE PRECISION,           
            name VARCHAR(255)               
        );
        , parameters: None
[2025-02-26T21:11:41.538+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=api_to_db_dag, task_id=create_station_informations_table, execution_date=20250226T211140, start_date=20250226T211141, end_date=20250226T211141
[2025-02-26T21:11:41.577+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-02-26T21:11:41.595+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
